packages/vite/src/node/config.ts

```diff
@@ -252,6 +252,11 @@ export interface SharedEnvironmentOptions {
    * Optimize deps config
    */
   optimizeDeps?: DepOptimizationOptions
+  /**
+   * Function to specify when modules should be considered built-in for the environment.
+   * (If not provided the node built-in modules are the only ones assumed as such)
+   */
+  isBuiltin?: (id: string) => boolean
```

Comment:
I think this should be moved under `resolve`

Reply:
ah yes definitely! thanks! moved 🙂👍